circleguard/loadable.py

```diff
@@ -663,6 +663,7 @@ class ReplayPath(Replay):
     def __init__(self, path, cache=None):
         self.log = logging.getLogger(__name__ + ".ReplayPath")
         self.path = path
+        self.hash = None
```

Comment:
is this hash being used for anything?

Reply:
If the api fails this is the only way to Identify a map. Same as how the username is kept even if we get the user id.